src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/Microsoft.DotNet.SourceBuild.SmokeTests.csproj

```diff
@@ -1,31 +1,124 @@
+      <SourceBuiltArtifactsPath Condition="'$(SourceBuiltArtifactsPath)' == ''">%(SourceBuiltArtifactsItem.Identity)</SourceBuiltArtifactsPath>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <!-- Required values -->
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).DotNetDirectory">
+        <Value>$(TargetDir)extracted-sdk</Value>
+      </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).TargetRid">
+        <Value>$(TargetRid)</Value>
+      </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).PortableRid">
+        <Value>$(PortableRid)</Value>
+      </RuntimeHostConfigurationOption>
+
+      <!-- Optional values -->
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).CustomPackagesPath">
+        <Value>$(SmokeTestsCustomSourceBuiltPackagesPath)</Value>
+      </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).ExcludeOmniSharpTests">
+        <Value>$(SmokeTestsExcludeOmniSharpTests)</Value>
+      </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).IncludeArtifactsSizeTests">
+        <Value>$(SmokeTestsIncludeArtifactsSizeTests)</Value>
+      </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).LicenseScanPath">
+        <Value>$(SmokeTestsLicenseScanPath)</Value>
+      </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).MsftSdkTarballPath">
+        <Value>$(MsftSdkTarballPath)</Value>
+      </RuntimeHostConfigurationOption>      
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).PoisonReportPath"
+                                      Condition="'$(EnablePoison)' == 'true'">
+        <Value>$(PoisonUsageReportFile)</Value>
+      </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).PrereqsPath">
+        <Value>$(SmokeTestsPrereqsPath)</Value>
+      </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).SdkTarballPath">
+        <Value>$(SdkTarballPath)</Value>
+      </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).SourceBuiltArtifactsPath">
+        <Value>$(SourceBuiltArtifactsPath)</Value>
+      </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).WarnOnLicenseScanDiffs">
+        <Value>$(SmokeTestsWarnOnLicenseScanDiffs)</Value>
+      </RuntimeHostConfigurationOption>
+      <RuntimeHostConfigurationOption Include="$(MSBuildProjectName).WarnOnSdkContentDiffs">
+        <Value>$(SmokeTestsWarnOnSdkContentDiffs)</Value>
+      </RuntimeHostConfigurationOption>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="CheckIfCreateSmokeTestPrereqsExistToPack"
```

Comment:
This can be removed along with the dependent targets.  This was replaced with mirroring the MSFT package feed and sharing that.